File: URI/1 - INICIANTE/Python/1044 - Multiplos.py
a, b = input().split()
a = int(a)
b = int(b)
if a < b:
    c = b % a
    if(c == 0):
        print('Sao Multiplos')
    else:
        print('Nao sao Multiplos')
else:
    c = a % b
    if (c == 0):
        print('Sao Multiplos')
    else:
        print('Nao sao Multiplos')

File: URI/1 - INICIANTE/C/1044 - Multiplos.c
#include <stdio.h>

int main()
{
	int a, b, c;
	
	scanf(" %d %d", &a, &b);
	
	if (a<b)
	{
		c = b%a;
	}else
	{
		c = a%b;
	}
	if(c==0)
	{
		printf("Sao Multiplos\n");
	}else{
		printf("Nao sao Multiplos\n");
	}
	return 0;
}
